src/etaoin/api.clj

```diff
@@ -710,11 +723,21 @@
     [:session (:session @driver) :element el :click]
     nil _))
 
-(defn click
+(declare wait)
+
+(defmulti click
```

Comment:
@jwkoelewijn in 0.2.0-snapshot, I implemented `click` as a multimethod, so you are welcome to extend it locally. I believe that 1 second wait for Safari requirement depends on your local env, OS, etc.